pkg/terminal/proxy.go

```diff
@@ -123,9 +123,13 @@ func (p *Proxy) HandleProxy(user *auth.User, w http.ResponseWriter, r *http.Requ
 			return
 		}
 
-		userId = string(userInfo.Status.UserInfo.UID) // TODO: are we cool rewriting this here?
+		userId = userInfo.Status.UserInfo.UID
 		if userId == "" {
-			userId = userInfo.Status.UserInfo.Username
+			// uid is missing. it must be kube:admin
```

Comment:
this breaks oidc where UID does not have to be set and username is not going to be `kube:admin`